org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -197,11 +197,11 @@ public static Object get(Object model, String name) throws IllegalAccessExceptio
 		}
 		return null;
 	}
-	
+
 	private static Object get(Object model, Class<?> clazz, String name) throws IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)